src/main/java/doore/login/utils/AuthorizationExtractor.java

```diff
@@ -0,0 +1,37 @@
+package doore.login.utils;
+
+import static doore.login.exception.LoginExceptionType.INVALID_ACCESS_TOKEN;
+import static doore.login.exception.LoginExceptionType.NOT_FOUND_AUTHORIZATION_TOKEN;
+
+import doore.login.exception.LoginException;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpHeaders;
+
+public class AuthorizationExtractor {
+    private static final String ACCESS_TOKEN_TYPE = AuthorizationExtractor.class.getSimpleName() + ".ACCESS_TOKEN_TYPE";
+    private static final String BEARER_TYPE = "Bearer";
+
+    public static String extract(final HttpServletRequest request) {
+        final String authorizationHeader = request.getHeader(HttpHeaders.AUTHORIZATION);
+
+        validateAuthorizationHeader(authorizationHeader);
+
+        String authHeaderValue = authorizationHeader.substring(BEARER_TYPE.length()).trim();
+
```

Comment:
```suggestion
int commaIndex = authHeaderValue.indexOf(',');
int endIndex = (commaIndex > 0) ? commaIndex : authHeaderValue.length();
String authHeaderValue = authorizationHeader.substring(BEARER_TYPE.length(), endIndex).trim();
```

요렇게 하면 아래 If문 없앨수 있을 것 같습니다🤔

Reply:
이 부분은 `authHeaderValue`가 선언되기 전에 `commaIndex `쓰이고 있는 것 같아요...! 혹시 제가 잘못 이해했을 수도 있느니 회의시간에 이야기해보면 좋을 것 같아요!